go.mod

```diff
@@ -122,6 +122,15 @@ require (
 	k8s.io/utils v0.0.0-20221012122500-cfd413dd9e85
 )
 
+require (
```

Comment:
can you please clean this up and make sure there are only 2 `require` blocks here? The first block is the direct depedencies, the second one is the indirect ones.